src/Filter/AbstractDateFilter.php

```diff
@@ -49,43 +49,47 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
         $data['type'] = !isset($data['type']) || !is_numeric($data['type']) ? DateType::TYPE_EQUAL : $data['type'];
 
         // Some types do not require a value to be set (NULL, NOT NULL).
-        if (!$this->typeRequiresValue($data['type']) && !$data['value']) {
+        if (!$this->typeRequiresValue($data['type']) && !isset($data['value'])) {
             return;
         }
 
         switch ($data['type']) {
             case DateType::TYPE_EQUAL:
+                $this->active = true;
+
                 $this->applyTypeIsEqual($queryBuilder, $field, $data);
 
                 return;
 
             case DateType::TYPE_GREATER_THAN:
-                if (!\array_key_exists('value', $data) || !$data['value']) {
-                    return;
-                }
+                $this->active = true;
 
                 $this->applyTypeIsGreaterThan($queryBuilder, $field, $data);
 
                 return;
 
             case DateType::TYPE_LESS_EQUAL:
-                if (!\array_key_exists('value', $data) || !$data['value']) {
```

Comment:
With the above check (`if (!$this->typeRequiresValue($data['type']) && !isset($data['value']))`) this is not required. 

It checks if the type is `TYPE_NULL` or `TYPE_NOT_NULL`, if it's not, it checks that the `$data['value']` exists.